src/index.js

```diff
@@ -39,7 +39,8 @@ RootControlComponent.prototype.rootUpdate = function(_control) {
 extend(RootControlComponent, BaseComponent);
 
 function createNode({ el, nodeProps, component }) {
-  const comp = Object.create(component.component.prototype, nodeProps) || new NodeComponent(nodeProps);
```

Comment:
Can u explain what happens, changes here?

Reply:
Resolved in later commit.